snscrape/modules/vkontakte.py

```diff
@@ -56,6 +56,12 @@ def get_items(self):
 			logger.error('Private profile')
 			return
 
+		profile_deleted = soup.find('h5', class_ = 'profile_deleted_text')
+		if profile_deleted:
+			# Unclear what this state represents, so just log website text.
+			logger.error(f'{profile_deleted.text}')
```

Comment:
This should be `logger.error(profileDeleted.text)`. No need to use an f-string if no formatting is happening.